### PR TITLE
Add check_skip example

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR follows https://github.com/mstachniuk/ci-skip/pull/4#issuecomment-730481614 and adds a basic example on `.github/workflows` plus some explanations in the `Readme`.

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
Should I add the trigger:

```
  pull_request:
    branches: [ master ]
```

in `.github/workflows/fail-safe-commit-filter-separator-should-success.yml` ?

Let me know what you think @mstachniuk

File: Readme.adoc [hunk 1]
```diff
@@ -116,6 +116,38 @@ If you want to use multiple filters at once, it's possible to separate them with
 [...]
 ----
 
+=== Job to check skip message
```
Comment (larsoner) [inline Readme.adoc:119]: Personally I would put this one at the top, above all other methods. It behaves most similarly to how `ci skip` does on all other platforms with fewer drawbacks.

Comment (mstachniuk) [reply Readme.adoc:119]: Thank you for your comment. Let's merge as it is for now. The example is a little bit long what can be a problem for first-time users.

**Comment by mstachniuk** [comment]
Let's merge as it is. I will change in all yamls to:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```
what is the best setup in my opinion.

**Pull request merged**